packages/data-context/src/actions/WizardActions.ts

```diff
@@ -323,15 +324,15 @@ interface E2eScaffoldOpts {
 
 const E2E_SCAFFOLD_BODY = (opts: E2eScaffoldOpts) => {
   return `
-    e2e: {
-      supportFile: 'cypress/support/e2e.${opts.lang}',
-      specPattern: '${getDefaultSpecPatterns().e2e}',
-      viewportHeight: 660,
-      viewportWidth: 1000,
-      setupNodeEvents(on, config) {
-        //
-      },
-    }
+  e2e: {
+    supportFile: 'cypress/support/e2e.${opts.lang}',
+    specPattern: '${getDefaultSpecPatterns().e2e}',
+    viewportHeight: 660,
+    viewportWidth: 1000,
```

Comment:
```suggestion
```

After meeting w/ @brian-mann yesterday, he pointed out that since these are the default values, we shouldn't actually need to specify them in the config file we generate for the setup